TESTS/netsocket/tls/tlssocket_handshake_invalid.cpp

```diff
@@ -28,12 +28,19 @@ using namespace utest::v1;
 
 void TLSSOCKET_HANDSHAKE_INVALID()
 {
+    const int https_port = 443;
     SKIP_IF_TCP_UNSUPPORTED();
     TLSSocket sock;
     TEST_ASSERT_EQUAL(NSAPI_ERROR_OK, sock.open(NetworkInterface::get_default_instance()));
     TEST_ASSERT_EQUAL(NSAPI_ERROR_OK, sock.set_root_ca_cert(tls_global::cert));
-    TEST_ASSERT_EQUAL(NSAPI_ERROR_AUTH_FAILURE,
-                      sock.connect("google.com", 443)); // 443 is https port.
+    TEST_ASSERT_EQUAL(NSAPI_ERROR_AUTH_FAILURE, sock.connect("www.mtvuutiset.fi", https_port));
```

Comment:
Use a local/CI host.

Reply:
Sorry, I didn't understand your comment. Could you please describe more?

Reply:
@AriParkkila , was there something you requested for this?

Reply:
How about using e.g. `example.com` here instead of `www.mtvuutiset.fi`.

Reply:
Example.com does not work either. Probably uses some CA root already included in the BG96 modem. But I removed mtvuutiset.fi as well. I think those other handshake tests verifies the needed functionality.